packages/winsvc-windows.0.1.0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-winsvc"
+build: [
+  ["./configure" "--host=%{conf-gcc-windows:host}%" "--prefix=%{prefix}%" "OCAMLFIND_TOOLCHAIN=windows"]
+  [make]
+]
+install: [
+  ["env" "OCAMLFIND_TOOLCHAIN=windows"
+   "make" "install"]
+]
+remove: ["ocamlfind" "-toolchain" "windows" "remove" "winsvc"]
+depends: [
+  "ocaml-windows"
+  "ocamlfind" {build}
```

Comment:
`ocamlfind` is intentionally implied by `ocaml-windows`.

Reply:
Should still be indeed.